packages/components/pages/f-account-info/stories/AccountInfo.stories.js

```diff
@@ -14,7 +14,7 @@ export const AccountInfoComponent = () => ({
     components: { AccountInfo },
     props: {
     },
-    template: '<account-info />'
+    template: '<account-info is-auth-finished smart-gateway-base-url="/" />'
```

Comment:
You should be able to leave out the `/` on the url because the api provider sorts this out 👍 
```suggestion
    template: '<account-info is-auth-finished smart-gateway-base-url="" />'
```